sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ClientRetryPolicy.java

```diff
@@ -339,6 +339,10 @@ public com.azure.cosmos.implementation.RetryContext getRetryContext() {
         return BridgeInternal.getRetryContext(this.getCosmosDiagnostics());
     }
 
+    public boolean canUsePreferredLocations() {
```

Comment:
is this method only used in tests? -- then we should not add it - we should be able to use RefelctionUtils to examine the values

Reply:
Will remove it, thanks.

Reply:
I disagree - it is in implementation package - and it is useful to have the API. Not necessary to jump through reflection for it,